monkey_island/cc/main.py

```diff
@@ -48,6 +48,9 @@
     },
 ]
 
+INITIAL_USERNAMES = ['Administrator', 'root', 'user']
```

Comment:
I'd just add this to the default mongodb configuration (same as alive:true), but that's a separate PR/Issue.